### PR TITLE
Added zaakurl and due date to portaltask object

### PR DESCRIPTION
**Pull request by ivo-ritense**

### CODE REVIEW

File: zgw/portaaltaak/src/main/kotlin/com/ritense/portaaltaak/PortaaltaakPlugin.kt [hunk 1]
```diff
@@ -91,6 +93,7 @@ class PortaaltaakPlugin(
         @PluginActionProperty receiver: TaakReceiver,
         @PluginActionProperty identificationKey: String?,
         @PluginActionProperty identificationValue: String?,
+        @PluginActionProperty verloopDurationInDays: Long?,
```
Comment (Klaas-Ritense) [inline zgw/portaaltaak/src/main/kotlin/com/ritense/portaaltaak/PortaaltaakPlugin.kt:96]: This doesn't break when an implementation already has this plugin-action configured?

Comment (ivo-ritense) [reply zgw/portaaltaak/src/main/kotlin/com/ritense/portaaltaak/PortaaltaakPlugin.kt:96]: The tests already covered a null argument situation. I added another test to verify that the framework will pass in a null if the property is missing from the configuration. So this shouldn't break.

**Review by Klaas-Ritense — DISMISSED** [review]

**Review by Klaas-Ritense — APPROVED** [review]
